pom.xml

```
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.40</version>
+        <version>4.73</version>
```

Optional, but nice to use most recent parent pom release.
```suggestion
        <version>4.74</version>
```